packages/nouns-webapp/src/components/Bid/index.tsx

```diff
@@ -247,13 +257,35 @@ const Bid: React.FC<{
             <span className={classes.customPlaceholder}>ETH</span>
           </>
         )}
-        <Button
-          className={auctionEnded ? classes.bidBtnAuctionEnded : classes.bidBtn}
-          onClick={auctionEnded ? settleAuctionHandler : placeBidHandler}
-          disabled={isDisabled}
-        >
-          {bidButtonContent.loading ? <Spinner animation="border" /> : bidButtonContent.content}
-        </Button>
+        {
+          !auctionEnded ? (
+              <Button
+                className={auctionEnded ? classes.bidBtnAuctionEnded : classes.bidBtn}
+                onClick={auctionEnded ? settleAuctionHandler : placeBidHandler}
+                disabled={isDisabled}
+              >
+                {bidButtonContent.loading ? <Spinner animation="border" /> : bidButtonContent.content}
+              </Button>
+          ) 
+          : (
+            <>
+              <Button
+                className={classes.bidBtnAuctionEnded}
+                onClick={fomoNounsBtnOnClickHandler}
+              >
+                Vote for the next Noun ⌐◨-◨
+            </Button>
+            {/* Only show force settle button if wallet connected */}
+            {isWalletConnected ? (<p className={classes.emergencySettleWrapper}>
```

Comment:
Got it. Will do. Wasn't sure if it was ok to use an a tag outside of the `Link` component. 

Def a low priority thing but it actually might be helpful from a dev velocity point of view to have a rough style guide.

Reply:
Oh wait ... I read this too quickly. 

The reason I opted for button here is that using an `a` tag requires an href, which we don't actually need in this case. The linter was complaining about having a `#` as the href.  Using a button styled as a link seemed to be the best solution based on what I read.

Reply:
Addressed the 

> Since the fallback is to do nothing, it might make sense to use && here instead of ? :. e.g.:

comment

Reply:
ah yes ` If you cannot provide an href, but still need the element to resemble a link, use a button and change it with appropriate styles.` missed that warning. all good!

Reply:
> <img alt="Screen Shot 2022-01-07 at 11 17 09 AM" width="626" src="https://user-images.githubusercontent.com/12565062/148573598-7d0db167-10f2-49d3-b283-105588d90bc4.png">

Since the button is black and text is white, it might look better to use the alternate ASCII: ⌐◧-◧